packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../global/functions';
+
+.usa-label {
+  font-size: rem-override(1.06rem);
+}
+
+.usa-input {
```

Comment:
Do we really want all of these styles to be loaded into every single component? I'm assuming the approach is to import `@import '../../global/formation_overrides';` into all USWDS v3 component variations? But a component like banner would not need these styles.